app/services/induction/change_mentor.rb

```diff
@@ -24,22 +24,18 @@ def initialize(induction_record:, mentor_profile: nil)
     @mentor_profile = mentor_profile
   end
 
-  def send_training_materials
-    MentorMailer.with(
-      mentor_profile:,
-      ect_name: induction_record.participant_profile.user.full_name,
-      cip_materials_name: cip_materials.name,
-      sit_name:,
-    ).training_materials
-     .deliver_later
+  def amend_mentor_cohort
+    Induction::AmendParticipantCohort.new(participant_profile: mentor_profile,
```

Comment:
Should we be using this service to amend a mentor profile? If it's generic enough to amend any type of profile or certain types of profile perhaps we should rename it to reflect that?